MIP/mip-40/README.md

```diff
@@ -0,0 +1,83 @@
+# MIP-40: *AB-FFS Decoupled*
```

Comment:
this is not FFS it is Postconfirmation (MCR is an instantiation of postconfirmation)

FFS = Postconfirmation AND L2-confirmation, which are potentially two different protocols.

Reply:
see https://github.com/movementlabsxyz/MIP/pull/37

Reply:
https://github.com/movementlabsxyz/MIP/blob/4ac00e0f8fc2b06ba544fd2e6cdc0d14bcd52a91/MIP/mip-34/README.md?plain=1#L69

Above are you referring to the title? This is a shorthand for referring to a system that uses both the Atomic Bridge and Fast Finality Settlement.